docs/articles/documentation/test-api/actions/action-options.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: docs
+title: Action Options
+permalink: /documentation/test-api/actions/action-options.html
+---
+# Action Options
+
+This topic describes three types of [test action](index.md) options.
+
+* [Mouse Action Options](#mouse-action-options)
+* [Click Action Options](#click-action-options)
+* [Typing Action Options](#typing-action-options)
+
+## Mouse Action Options
+
+Provide additional parameters for a mouse action.
+
+```js
+{
+    ctrl: false | true,
+    alt: false | true,
+    shift: false | true,
+    meta: false | true,
+    offsetX: Number,
+    offsetY: Number
+}
+```
+
+Parameter                      | Type    | Description                                                                                                                               | Default
+------------------------------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------
+`ctrl`, `alt`, `shift`, `meta` | Boolean | Indicate which modifier keys are to be pressed during the mouse action.                                                                   | `false`
+`offsetX`, `offsetY`           | Number  | Mouse pointer coordinates relative to the top-left corner of the target element. Define a point where the action is performed or started. | The center of the target element.
+
+Mouse action options are used in the [t.drag](drag-element.md#drag-an-element-by-an-offset),
+[t.dragToElement](drag-element.md#drag-an-element-onto-another-one) and [t.hover](hover.md) actions.
+
+## Click Action Options
+
+Provide additional parameters for a click action.
+
+```js
+{
+    ctrl: false | true,
+    alt: false | true,
+    shift: false | true,
+    meta: false | true,
+    offsetX: Number,
+    offsetY: Number,
+    caretPos: Number
+}
+```
+
+Parameter                      | Type    | Description                                                                                                                               | Default
+------------------------------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------
+`ctrl`, `alt`, `shift`, `meta` | Boolean | Indicate which modifier keys are to be pressed during the mouse action.                                                                   | `false`
+`offsetX`, `offsetY`           | Number  | Mouse pointer coordinates relative to the top-left corner of the target element. Define a point where the action is performed or started. | The center of the target element.
```

Comment:
IMHO we need to say that `offsetX` and `offsetY` must be integer, otherwise they will not pass validation.